docs/source/tutorial/representations.rst

```diff
@@ -69,23 +69,21 @@ predictions for entities not seen during training.
 
     from pykeen.pipeline import pipeline
     from pykeen.datasets import get_dataset
-    from pykeen.nn.representation import EmbeddingSpecification, LabelBasedTransformerRepresentation
+    from pykeen.nn.init import LabelBasedInitializer
     from pykeen.models import ERModel
 
     dataset = get_dataset(dataset="nations")
-    entity_representations = LabelBasedTransformerRepresentation.from_triples_factory(
-        triples_factory=dataset.training,
-    )
+    entity_representations = LabelBasedInitializer.from_triples_factory(dataset.training).as_embedding()
```

Comment:
This changed the functionality:

before we have the encoder's params be part of the model and thus receive updates during training. Now, we switched to a fixed embedding with pre-trained embeddings.

Reply:
Oops I messed that up